modules/dists/BlockDist.chpl

```diff
@@ -235,13 +235,7 @@ The ``blockDist`` initializer is defined as follows:
 
     proc blockDist.init(
       boundingBox: domain(?),
-      targetLocales: [] locale  = Locales,
-      dataParTasksPerLocale     = // value of dataParTasksPerLocale config const,
-      dataParIgnoreRunningTasks = // value of dataParIgnoreRunningTasks config const,
-      dataParMinGranularity     = // value of dataParMinGranularity config const,
```

Comment:
There's a sentence referring to the dataPar variables [above](https://github.com/chapel-lang/chapel/pull/23345/files#diff-23d7257f759fdcebf3095f1855021eebd706f2a77b241bc5a6d5e9777ff8990aR224-R227) which should probably simply be removed for simplicity.